README.md

```diff
@@ -55,7 +55,7 @@ This software currently supports Ubuntu 18.04 LTS.
 The server version is sufficient, see https://ubuntu.com/download/server.
 *Note: The Windows Subsystem for Linux (WSL) currently does <u>not</u> support CUDA.*
 
-When going beyond simple tests, model training requires CUDA hardware acceleration (the network loader does not require CUDA).
+When going beyond simple models, model training does not work well without CUDA hardware acceleration (the network loader does not require CUDA, and very simple models can be trained on the CPU only).
```

Comment:
I suggest:"  ... and very simple models can still be trained on the CPU."

Reply:
Thanks, I clarified this.